README.md

```diff
@@ -22,3 +22,25 @@ Tested on Ubuntu 18.04 and Debian 10.
 * Put all these files in `/var/www/html`
 * Run `sudo setup.sh`
 * Visit http://localhost in your browser
+
+FAQ
+----
+**Can you delete it when you are done with it?**
+
+Yes. For any wiki you create, you will see a `Delete` link in the `Action` column of this table: https://patchdemo.wmflabs.org/
+
+**How long do the Patch demo wiki instances last for?**
+
+There is no definitive time after which wikis will automatically be deleted. With this said, we make no guarantees about how long they will continue to exist. A Patch demo wiki you've created could be deleted if we need to free up disk space to create space for new ones.
+
+**Can Patch demo wikis be named?**
+
+Wikis can not been named *within* Patch demo. Patch demo wikis currently inherit the name of the patch on which they are built.
+
+**Is it possible to add extensions that are in development?**
+
+#TODO
+
+**Is it possible to add patches for extension not just core? And skins?**
+
```

Comment:
Yes, any extension or skin that is currently installed (see repositories.txt) can have a patch applied.

Reply:
I would phrase it along the lines of:

Yes, patches for many extensions and skins are supported (mostly those included in MediaWiki releases, or enabled on all Wikimedia wikis), as well as Parsoid. Check out the list under "Choose extensions to enable" in the interface.

Reply:
Done.